components/mixins/experience-event/experienceevent-claim-process.schema.json

```diff
@@ -24,42 +22,37 @@
         "xdm:disputes": {
           "type": "object",
           "properties": {
```

Comment:
Do we need to keep the nesting "disputes" and "claimProcess". We talked about removing nesting with the same property name both as parent and child.

Reply:
This nesting should not have been included with this branch - I've updated it accordingly.